lib/reply.js

```diff
@@ -409,7 +409,7 @@ function onSendEnd (reply, payload) {
     // we cannot send a content-length for 304 and 204, and all status code
     // < 200.
     // For HEAD we don't overwrite the `content-length`
-    if (statusCode >= 200 && statusCode !== 204 && statusCode !== 304 && req.method !== 'HEAD') {
+    if ((statusCode < 200 || statusCode === 204 || statusCode === 304) && req.method !== 'HEAD') {
```

Comment:
This change will allow sending `content-length` for the `status code < 200 || 204 || 304`
Setting it to zero is also sending the `content-length` header

Reply:
sorry, if i misunderstood the code, but to me it looks like, for `status code < 200 || 204 || 304` it will set the `content-length` header to 0, so its overwriting the `content-header`

Reply:
ahhh now i understand, okay this change makes no sense, sorry :v: